Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/Vehicle.hs

```diff
@@ -70,6 +71,15 @@ updateVehicleRec vehicle = do
       ]
     where_ $ tbl ^. VehicleTId ==. val (toKey vehicle.driverId)
 
+updateVehicleName :: Maybe Text -> Id Person -> SqlDB ()
+updateVehicleName vehicleName driverId = do
+  Esq.update $ \tbl -> do
+    set
+      tbl
+      [ VehicleVehicleName =. val vehicleName
+      ]
```

Comment:
What is `VehicleName`? Also, update `updatedAt` column

Reply:
As discussed already , this vehicle name is the name for the vehicle given by the driver, use case was to improve the ux